### PR TITLE
Describe metrics (look up metric name by key-value pair)

### PR DESCRIPTION
**Pull request by Nathan-Fenner**

### CODE REVIEW

File: query/language.peg [hunk 1]
```diff
@@ -267,5 +270,5 @@ PAREN_OPEN <-  _ "(" _
 PAREN_CLOSE <- _ ")" _
 COMMA <-       _ "," _
 _  <-          SPACE* # Optional spaces
-__ <-          SPACE+ # Required spaces
+__ <-          !ID_CONT SPACE* # "Required" spaces
```
Comment (jeeyoungk) [inline query/language.peg]: let's add test for it.

File: main/static/index.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
 		<div ng-show="queryResult.name === 'select'" id="chart-div-container">
 			<div id="chart-div"></div>
 		</div>
-		<table ng-show="queryResult.name === 'describe all'" class="table table-striped col-xs-12">
+		<table ng-show="queryResult.name === 'describe all' || queryResult.name === 'describe metrics' || queryResult.name === 'describe'" class="table table-striped col-xs-12">
```
Comment (jeeyoungk) [inline main/static/index.html]: let's move this condition to a javascript function, `$scope.isTabular()`

File: main/ui.go [hunk 3]
```diff
@@ -28,6 +28,11 @@ func main() {
 	common.SetupLogger()
 
 	apiInstance := common.NewAPI()
-	backend := backend.NewSequentialMultiBackend(blueflood.NewBlueflood(*common.BluefloodUrl, *common.BluefloodTenantId))
+	bluefloodConfig := blueflood.BluefloodClientConfig{
```
Comment (jeeyoungk) [inline main/ui.go]: sorry i screwed this one up :/

File: query/language.peg (hunk 1 above)
Comment (Nathan-Fenner) [inline query/language.peg]: New tests in `command_test.go`

**Comment by jeeyoungk** [comment]
LGTM

**Pull request merged**